### PR TITLE
rosa hcp upgrade to 4.14

### PR DESCRIPTION
**Pull request by xinredhat**

### CODE REVIEW

**Comment by Roming22** [comment]
There's a typo in the commit message, version should be `4.14`.
My understanding is that the tests should go through without having to merge the PR first, is that correct?
If so, ping me again when the PR is working as expected.

**Comment by xinredhat** [comment]
@Roming22 thank you for catching that error. Yes, I am investing why pipeline-service cannot install properly on OCP 4.14

**Comment by xinredhat** [comment]
pipeline-service cannot install properly on OCP 4.14 because there is no minio-operator on OCP 4.14.

**Comment by xinredhat** [comment]
waiting for fix of https://github.com/minio/operator/issues/1875

**Comment by xinredhat** [comment]
https://github.com/minio/operator/issues/1875 is reopened, lets wait for its fix

**Comment by xinredhat** [comment]
It failed at installing postgresql , I am investigating it

File: operator/test/test.sh [hunk 1]
```diff
@@ -359,9 +359,9 @@ test_results() {
       QUERY_URL="https://$RESULT_ROUTE/apis/results.tekton.dev/v1alpha2/parents/${LOG_PATH}"
       QUERY_CMD[6]="${QUERY_URL}"
       LOGS_RESULT=$("${QUERY_CMD[@]}" 2>/dev/null)
-      LOGS_OUTPUT=$(echo "$LOGS_RESULT" | jq -r ".result.data | @base64d")
+      # LOGS_OUTPUT="$LOGS_RESULT"
```
Comment (enarha) [inline operator/test/test.sh:362]: Is that a temporary thing? If it's not needed, we better remove it.

**Review by enarha — COMMENTED** [review]
Apart from a nit, the rest is LGTM. I'll encourage you though to add more descriptive description :-) Some changes does not look related to the actual OCP upgrade and further it could be indicated if they are temporary or long term changes.

**Review by Roming22 — APPROVED** [review]

**Pull request merged**